src/test/java/org/fisco/bcos/web3j/abi/TypeEncoderTest.java

```diff
@@ -257,4 +259,79 @@ public void testEmptyArray() {
                 TypeEncoder.encodeDynamicArray(array),
                 is("0000000000000000000000000000000000000000000000000000000000000000"));
     }
+
+    @Test
+    public void throwsIfMaxOfU256() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/app/fisco/web3sdk/pullRequest?prid=4732356)